tests/providers/google/cloud/operators/test_translate_speech_system.py

```diff
@@ -18,8 +18,8 @@
 
```

Comment:
This file should be deleted - check with other PRs
test_SERVICE_system.py should be deleted ;)